pkg/provider/github/status.go

```diff
@@ -150,6 +156,13 @@ func (v *Provider) updatePipelineRunWithCheckRunID(ctx context.Context, tekton v
 		if err != nil {
 			return err
 		}
+		// temp fix: wait for tekton.dev/pipeline label to avoid race condition
+		// https://github.com/openshift-pipelines/pipelines-as-code/issues/786
+		if _, ok := pr.Labels["tekton.dev/pipeline"]; !ok {
+			v.Logger.Infof("PipelineRun %v/%v don't have tekton label yet, waiting for it", pr.GetNamespace(), pr.GetName())
+			time.Sleep(2 * time.Second)
```

Comment:
are we sure in 2 seconds we get that label :thinking: 

Or else shall we add waiting mechanism here to wait continuously untill we see that label

Reply:
ah i see 
:+1: